specs/protocol/abi.md

```diff
@@ -244,17 +311,25 @@ Below is a list of the JSON ABI formats for each available type:
 Given the following ABI declaration:
 
 ```rust
-enum MyEnum {
+enum MyEnum<V> {
     Foo: u64,
     Bar: bool,
 }
-struct MyStruct {
+struct MyStruct<T, U> {
```

Comment:
Maybe the `U` and `V` types could be used by the components themselves? ie have `Foo: V`. This would make the example more clear.

Reply:
I think you're right. I wanted to emphasize that the generic types don't really need to be used by the data memebers (e.g. Vec<T>`). However, maybe a good compromise is to use one generic type and not use the other.

Reply:
Yes I think that's a good idea!

Reply:
Done i n https://github.com/FuelLabs/fuel-specs/pull/382/commits/fdf55f4b6b0e3073a1da5f8ea5b28cd8f272d768